nengo/spa/tests/test_spa.py

```diff
@@ -9,12 +9,19 @@ def test_spa_verification(Simulator, seed, plt):
     d = 16
 
     model = spa.SPA(seed=seed)
+
     # build a normal model that shouldn't raise a warning
     with model:
         model.buf = spa.Buffer(d)
         model.input_node = spa.Input(buf='B')
         # make sure errors aren't fired for non-spa modules
         prod = nengo.networks.Product(10, 2)  # noqa: F841
+        model.int_val = 1
```

Comment:
It is testing that attribute assignment with non-spa module stuff still works without a warning?